addon/components/au-dropdown.js

```diff
@@ -2,8 +2,30 @@ import Component from '@glimmer/component';
 import { deprecate } from '@ember/debug';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
+import { modifier } from 'ember-modifier';
+
+import FloatingUiModifier from '@appuniversum/ember-appuniversum/private/modifiers/floating-ui';
 
 export default class AuDropdown extends Component {
+  @tracked referenceElement = undefined;
+  @tracked arrowElement = undefined;
+
+  floatingUi = FloatingUiModifier;
+
+  reference = modifier(
+    (element) => {
+      this.referenceElement = element;
```

Comment:
Since we now have a reference to the button I think we no longer need to query for it here either:

https://github.com/appuniversum/ember-appuniversum/blob/363616290579144ddcaf2760ce50421aa84a3e40/addon/components/au-dropdown.js#L26

Reply:
Does this work when you have multiple `AuDropdown` instances on the same page (it will always select the first match)? Or is the query only executed within the scope of the component?

Reply:
No, I think you are right. I should have added some sort of unique-id to the selector I think. Good we can replace it with the modifier now 😄. Not sure why I didn't do it like that in the first place.